tests/unittests/CMakeLists.txt

```diff
@@ -88,9 +88,10 @@ else()
       WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
       TEST_PREFIX "UT_" # make it possible to filter easily with -R ^UT
       TEST_SPEC ${filter_tests} # discover only tests that are known to not fail
-      DL_PATHS ${CMAKE_CURRENT_BINARY_DIR}:${PROJECT_BINARY_DIR}/src:${PROJECT_BINARY_DIR}/tests:$<TARGET_FILE_DIR:ROOT::Tree>:$<$<TARGET_EXISTS:SIO::sio>:$<TARGET_FILE_DIR:SIO::sio>>:$ENV{LD_LIBRARY_PATH}
       PROPERTIES
         ENVIRONMENT
-        PODIO_SIOBLOCK_PATH=${CMAKE_CURRENT_BINARY_DIR}
+        PODIO_SIOBLOCK_PATH=${PROJECT_BINARY_DIR}/tests
+        ENVIRONMENT
```

Comment:
Is this second `ENVIRONMENT` necessary?

Reply:
Having a look at the comment, I think the `SKIP_CATCH_DISCOVERY` option can be removed; the point of that part is to check if the cmake version is less than 3.22 because in principle using `DL_PATH` is needed and this is supported only for 3.22 and above. But, it is actually not needed, because duplicating the `ENVIRONMENT` keyword works (it's described in the Catch2 issue).